cache-ignite/src/test/groovy/io/micronaut/ignite/IgniteClientAsyncCacheSpec.groovy

```diff
@@ -14,7 +14,7 @@ import spock.lang.Shared
 class IgniteClientAsyncCacheSpec extends AbstractAsyncCacheSpec {
 
     @Shared
-    GenericContainer ignite = new GenericContainer("apacheignite/ignite:2.8.0")
+    GenericContainer ignite = new GenericContainer("apacheignite/ignite:2.8.1")
```

Comment:
One trick you can use to avoid hardcoding the version here, and using the same one as the project, is to pass it from Gradle. An example:

https://github.com/micronaut-projects/micronaut-elasticsearch/blob/master/build.gradle#L35

Which then you would reference in a test as:

https://github.com/micronaut-projects/micronaut-elasticsearch/blob/master/elasticsearch/src/test/groovy/io/micronaut/elasticsearch/health/ElasticsearchHealthIndicatorSpec.groovy#L39